### PR TITLE
tools: improve the Binlog Slave Client doc

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/docs-cn/pull/938
@lilin90 @WangXiangUSTC PTAL

### CODE REVIEW

File: tools/binlog-slave-client.md [hunk 1]
```diff
@@ -87,41 +88,41 @@ message TableMutation {
...
 message DMLData {
-  // tables contains all the table changes.
+  // `tables` 包含一个表的所有数据变更
   repeated Table tables = 1;
 }
 
 // DDLData 保存 DDL 的信息
 message DDLData {
-  // the current database use
+  // 当前数据库的使用
```
Comment (WangXiangUSTC) [inline tools/binlog-slave-client.md]: 当前使用的数据库

File: tools/binlog-slave-client.md [hunk 2]
```diff
@@ -87,41 +88,41 @@ message TableMutation {
...
   optional string schema_name = 1;
-  // the relate table
+  // 相关表
   optional string table_name = 2;
-  // ddl_query is the original ddl statement query.
+  // `ddl_query` 是最初的 DDL 语句 query
```
Comment (WangXiangUSTC) [inline tools/binlog-slave-client.md]: 最初改成原始比较好

File: tools/binlog-slave-client.md [hunk 3]
```diff
@@ -140,7 +141,7 @@ TiDB-Tools 项目中提供了用于读取 Kafka 中 binlog 数据的 Driver，
 
 Driver 项目地址：[Binlog Slave Driver](https://github.com/pingcap/tidb-tools/tree/master/tidb_binlog/driver)。
 
-注：
+**注意：**
```
Comment (lilin90) [inline tools/binlog-slave-client.md]: Please add the note format.

**Comment by CaitinChen** [comment]
@lilin90 @WangXiangUSTC Thanks. PTAL again.

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

**Comment by CaitinChen** [comment]
@WangXiangUSTC PTAL again

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**